src/operator/nn/mkldnn/mkldnn_batch_norm-inl.h

```diff
@@ -27,69 +27,72 @@
 #define MXNET_OPERATOR_NN_MKLDNN_MKLDNN_BATCH_NORM_INL_H_
 
 #if MXNET_USE_MKLDNN == 1
-#include <vector>
-#include <utility>
 #include <mkldnn.hpp>
+#include <utility>
+#include <vector>
 #include "../batch_norm-inl.h"
-#include "./mkldnn_ops-inl.h"
 #include "./mkldnn_base-inl.h"
+#include "./mkldnn_ops-inl.h"
 
-#define VARIANCE_TO_INVSTD(__var$,    __eps$)   (1.0/std::sqrt((__var$) + DType(__eps$)))
-#define INVSTD_TO_VARIANCE(__invstd$, __eps$)   ((1.0 / ((__invstd$) * (__invstd$))) - (__eps$))
+#define VARIANCE_TO_INVSTD(__var$, __eps$) \
+  (1.0 / std::sqrt((__var$) + DType(__eps$)))
+#define INVSTD_TO_VARIANCE(__invstd$, __eps$) \
+  ((1.0 / ((__invstd$) * (__invstd$))) - (__eps$))
 namespace mxnet {
 namespace op {
 
-typedef mkldnn::batch_normalization_forward::primitive_desc     t_bn_f_pdesc;
-typedef mkldnn::batch_normalization_forward::desc               t_bn_f_desc;
-typedef mkldnn::batch_normalization_backward::primitive_desc    t_bn_b_pdesc;
-typedef mkldnn::batch_normalization_backward::desc              t_bn_b_desc;
+typedef mkldnn::batch_normalization_forward::primitive_desc t_bn_f_pdesc;
```

Comment:
Could we left original alignment here ?

Reply:
I have noticed that alignment is still lost here.

Reply:
There's a special param that setting it up. Unfortunately, it doesn't match our requirements in some cases. A preferable case is keeping the name of a variable anchored to a type rather than putting variables one by one.